### PR TITLE
CI: Add ubuntu:bionic using docker.

### PR DESCRIPTION
**Pull request by fghoussen**
## Pull request purpose

CI: Add ubuntu:bionic using docker.

### CODE REVIEW

File: PARPACK/SRC/BLACS/Makefile.am [hunk 1]
```diff
@@ -20,4 +20,4 @@ libparpack@LIBSUFFIX@_la_LIBADD = \
 	$(top_builddir)/PARPACK/UTIL/BLACS/libparpackutil.la \
 	$(top_builddir)/SRC/libarpack@LIBSUFFIX@.la \
 	$(LAPACK_LIBS) $(BLAS_LIBS)
-libparpack@LIBSUFFIX@_la_LDFLAGS = -no-undefined -version-info 3:0:1
+libparpack@LIBSUFFIX@_la_LDFLAGS = -no-undefined -version-info 2:1:0
```
Comment (sylvestre) [inline PARPACK/SRC/BLACS/Makefile.am]: why this change here?

Comment (fghoussen) [reply PARPACK/SRC/BLACS/Makefile.am]: Just trying/testing ! Reverting this.
Is that normal that versions are not the same in cmake/autotools ?

**Comment by fghoussen** [comment]
no idea. @turboencabulator : any idea ?

**Comment by turboencabulator** [comment]
@fghoussen I assume you're referring to #204?

**Comment by fghoussen** [comment]
@sylvestre : rebased + update changelog
